pkg/baseline/packageIndex.json

```diff
@@ -9,8 +9,9 @@
       "BaselineVersion": "4.4.0",
       "InboxOn": {},
       "AssemblyVersionInPackageVersion": {
-        "4.1.1.0": "4.4.0",
-        "4.1.2.0": "4.4.0"
+        "4.1.1.0": "4.5.0",
```

Comment:
I see. Does this affect the building of package or is it only for validation? If it's former, we should make the 4.1.1.0->4.3.0 mapping change in release/2.0.0 branch too, correct?